Sources/TuistCache/Utilities/XCFrameworkBuilder.swift

```diff
@@ -86,7 +86,7 @@ public final class XCFrameworkBuilder: XCFrameworkBuilding {
             let deviceArchivePath = temporaryPath.appending(component: "device.xcarchive")
             let deviceArchiveObservable = xcodeBuildController.archive(projectTarget,
                                                                        scheme: scheme,
-                                                                       clean: true,
+                                                                       clean: false,
```

Comment:
Since we are using a temporary derived data directory that gets destroyed after building the xcframeworks, there's no need to pass the `clean` argument to `xcodebuild`.